### PR TITLE
Switch to open source UUID Generator

### PR DESCRIPTION
**Pull request by aarreedd**

### CODE REVIEW

**Comment by randomPoison** [comment]
Thanks for opening a pull request! Could you provide more rationale behind the changes here? I'm cool with changing the link to an open source tool, but I'm hesitant to merge a PR with no description.

**Comment by aarreedd** [comment]
There is no substantial difference between the two in terms of functionality. Just that the new one is open source and if I have the option I would rather use a tool that is open.

**Comment by randomPoison** [comment]
Fair enough! I'm all for pointing users to open source options where possible 😁

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ struct MyType;
 While the derive handles the tedious work of converting the UUID into a byte
 array suitable for use with the [`TypeUuid`] trait, you'll still need to
 generate a valid UUID in order to assign it to your type. To do so, we
-recommend using https://www.uuidgenerator.net, which provides a quick way
+recommend using https://www.uuidtools.com, which provides a quick way
```
Comment (randomPoison) [inline README.md]: One minor request: Point folks directly at the v4 UUID generator. It seems like a good idea to have folks use random UUIDs by default.

```suggestion
recommend using https://www.uuidtools.com/generate/v4, which provides a quick way
```

Comment (aarreedd) [reply README.md]: Makes sense.

**Pull request merged**